.php-cs-fixer.dist.php

```diff
@@ -14,6 +14,7 @@
 
 use PhpCsFixer\Config;
```

Comment:
Question for whole PR

Which actor writes to cache file? Worker or supervisor ?
We have some logic to limit parallel actors writing to cache and overriding one another on master branch

Reply:
Cache is used by the worker, which calls `Runner::fixFile()`. After merging #7769, rebasing this PR and building Docker image with cache support:

<img width="1728" alt="image" src="https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/assets/600668/d7159b6e-1c79-47bb-afd8-fcab001af83e">

- First run (sequential) creates cache
- second run (sequential) reuses cache info
- third run (parallel) reuses cache info

Unfortunately it works a bit worse when parallel run is used for creating cache:

<img width="1236" alt="image" src="https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/assets/600668/cfc79014-ba58-46e3-a851-012cb7d994db">

I will look at it what we can do about it. Good catch, I did not focus on it since cache is disabled for development 🤷‍♂️.

Reply:
@keradus it's resolved, cache is now fully supported in parallel analysis - details in [this commit](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7777/commits/12e893d5971be78462d14b1d0213d8509c2b6ab1).